docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -221,165 +228,166 @@ Additionally, the Zeebe Gateway should be configured to use an encrypted connect
 
 1. Configure your Zeebe Gateway Ingress to create a [Re-encrypt Route](https://docs.openshift.com/container-platform/latest/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration):
 
-```yaml
-zeebeGateway:
-  ingress:
-    grpc:
-      annotations:
-        route.openshift.io/termination: reencrypt
-        route.openshift.io/destination-ca-certificate-secret: camunda-platform-internal-service-certificate
-      className: openshift-default
-      tls:
-        enabled: true
-        secretName: camunda-platform-external-certificate
-```
+   ```yaml
+   zeebeGateway:
+     ingress:
+       grpc:
+         annotations:
+           route.openshift.io/termination: reencrypt
+           route.openshift.io/destination-ca-certificate-secret: camunda-platform-internal-service-certificate
+         className: openshift-default
+         tls:
+           enabled: true
+           secretName: camunda-platform-external-certificate
+   ```
```

Comment:
This PR seems pretty inconsistent. do you prefer indented codeblocks or non-indented codeblocks?
https://github.com/camunda/camunda-docs/pull/4224/files#diff-a2f46593f55132fad493be0c88dcd7ae6aa247e7afaf0bf345d2102fb9161661R76